templates/flagger.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: flagger
+  namespace: kubeaddons-flagger
+  labels:
+    kubeaddons.mesosphere.io/name: flagger
+  annotations:
+    sidecar.istio.io/inject: "false"
+    appversion.kubeaddons.mesosphere.io/flagger: "0.19.0"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.19.0"
+    docs.kubeaddons.mesosphere.io/flagger: "https://docs.flagger.app/"
+    values.chart.helm.kubeaddons.mesosphere.io/flagger: "https://raw.githubusercontent.com/weaveworks/flagger/0.19.0/charts/flagger/values.yaml"
+spec:
+  namespace: kubeaddons-flagger
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  cloudProvider:
+    - name: aws
+      enabled: true
```

Comment:
should we mark it as `false` by default as both `dispatch` and `istio` is disabled by default